### PR TITLE
repo, mappers and resolvers for equipment backend

### PR DESCRIPTION
**Pull request by smellslikekeenspirit**
plus `TrainingModule` model and mapper

### CODE REVIEW

File: server/src/mappers/equipment/Equipment.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Equipment } from "../../models/equipment/equipment";
+
+export function equipmentToDomain(raw: any): Equipment[] {
+  return raw.map((i: any) => {
```
Comment (MatthewGalan) [inline server/src/mappers/equipment/Equipment.ts:4]: Do we need to consider the possibility of `raw` being `null` or `undefined`?

File: server/src/mappers/equipment/EquipmentLabel.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { EquipmentLabel } from "../../models/equipment/equipmentLabel";
+
+export function equipmentLabelsToDomain(raw: any): EquipmentLabel[] {
+  return raw.map((i: any) => {
+    return singleEquipmentLabelToDomain(i);
+  })
```
Comment (MatthewGalan) [inline server/src/mappers/equipment/EquipmentLabel.ts:6]: Style suggestion:
``` js
return raw.map((i: any) => singleEquipementLabelToDomain(i));
```

**Review by MatthewGalan — APPROVED** [review]
Awesome work! I think we might need to add a reservation status at some point (pending, confirmed, cancelled), but this is a great start.

PS: Can we change the PR title so something more descriptive pls

File: server/src/mappers/equipment/Equipment.ts (hunk 1 above)
Comment (smellslikekeenspirit) [reply server/src/mappers/equipment/Equipment.ts:4]: @MatthewGalan it calls a function `singleEquipmentToDomain` which takes care of that

Comment (MatthewGalan) [reply server/src/mappers/equipment/Equipment.ts]: Right, but wouldn't we get an error like `Cannot read properties of undefined (reading 'map')` on line 4 first?

**Pull request merged**